jaxws-ri/tools/wscompile/src/test/java/com/sun/tools/ws/ant/WsAntTaskTestBase.java

```diff
@@ -167,6 +168,6 @@ protected boolean isAntPre18() {
     }
 
     static boolean is9() {
-        return System.getProperty("java.version").startsWith("9");
+        return ModuleFinder.ofSystem().find("java.xml.ws").isPresent();
```

Comment:
This doesn't look right.  I'm not sure what is9 is used for but I'm assuming it means JDK9 or later so maybe you should use Class.forName("java.lang.Module").

Reply:
These are src/test (testing classes), you will not call those in the production.